can-bus/json/FSM/FSM_rx.json

```
@@ -1,5 +1,6 @@
 {
     "messages": [
-        "BMS_Vitals"
+        "BMS_Vitals",
+        "Debug_CanModes"
     ]
 }
```

nit: missing blank line
```suggestion
}

```